File: example_1.py
import requests

def send_message(user_input):
    url = 'https://aelius.live/api/v1/chat'
    headers = {'Content-Type': 'application/json'}
    data = {'user_input': user_input}
    
    try:
        response = requests.post(url, headers=headers, json=data)
        response_data = response.json()
        display_message('Assistant', response_data['response'])
    except Exception as e:
        print(f"Error fetching AI response: {e}")

def display_message(who, message):
    # Implement this function based on how you want to display the message
    print(f"{who}: {message}")

# Example usage:
user_input_value = input("Type a message: ")
send_message(user_input_value)

File: example_1.js
const fetch = require('node-fetch');
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

async function sendMessage(userInput) {
    const url = 'https://aelius.live/api/v1/chat';
    const headers = {'Content-Type': 'application/json'};
    const data = {user_input: userInput};
    
    try {
        const response = await fetch(url, {
            method: 'POST',
            headers: headers,
            body: JSON.stringify(data)
        });
        const responseData = await response.json();
        displayMessage('Assistant', responseData.response);
    } catch (e) {
        console.error(`Error fetching AI response: ${e}`);
    }
}

function displayMessage(who, message) {
    console.log(`${who}: ${message}`);
}

// Example usage:
rl.question('Type a message: ', (userInputValue) => {
    sendMessage(userInputValue);
    rl.close();
});
